core/templates/css/oppia.css

```diff
@@ -1312,11 +1312,13 @@ span.sort-explorations-select .sort-order {
 /* Rules for the tiles in the creator dashboard. */
 
 .oppia-dashboard-intro-button {
-  background-color: #015c53;
+  background-color: #00645c;
   border-radius: 4px;
   color: #fff;
   font-family: "Roboto", Arial, sans-serif;
   font-size: 14px;
+  font-weight: 500;
```

Comment:
Why is this set?

Reply:
> @apurvbotle Please, add screenshots of all the changed buttons, so @mschanteltc can check if they look fine.

I confirmed in #11181 that the font weight should be 500 because it best represents that "medium" weight font. For some reason, setting it to "Medium" made it look bolder than I intended.